# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/Image.h
# Commit: 2e365eaea75b
# Full Hash: 2e365eaea75b925b2b5ff621634ba8f7d7fec00b
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-22 04:04:03
# Regressor Bug: 1568950
# File Overlap Count: 1
# Description:
#   Bug 1568950 - Part 5: Remove nsIDOMWindowUtils::forceUseCounterFlush; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D42818
# ==============================================================================

diff -r 1c1c4ca0bc14 -r 2e365eaea75b image/Image.h
--- a/image/Image.h	Wed Aug 21 10:29:14 2019 +0000
+++ b/image/Image.h	Wed Aug 21 12:52:19 2019 +0000
@@ -225,8 +225,6 @@
   virtual void SetHasError() = 0;
 
   virtual nsIURI* GetURI() const = 0;
-
-  virtual void ReportUseCounters() {}
 };
 
 class ImageResource : public Image {